#! /bin/sh /usr/share/dpatch/dpatch-run
## 05_fix_nonblocking_mode_typo.diff.dpatch by  <crimsun@localhost.localdomain>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Fix typo for nonblocking mode

@DPATCH@

# HG changeset patch
# User crimsun@localhost.localdomain
# Node ID 918f4dd50f7d88a77adb4e09d7822fa6cdf57ec8
# Parent  2d703ffa2dc05da9a2d724014d7b55a329c34505
Subject: [PATCH] [UBUNTU alsa-lib] Fix typo for nonblocking mode

UpstreamStatus: Applied, src/rawmidi/rawmidi_virt.c r1.7,
                         http://sourceforge.net/mailarchive/forum.php?thread_id=9644976&forum_id=33141

Jaroslav Kysela applied this patch that fixes a typo for nonblocking
mode.

Signed-off-by: Daniel T Chen <crimsun@ubuntu.com>

diff -r 2d703ffa2dc0 -r 918f4dd50f7d src/rawmidi/rawmidi_virt.c
--- a/src/rawmidi/rawmidi_virt.c	Sat Apr 29 17:34:32 2006 -0400
+++ b/src/rawmidi/rawmidi_virt.c	Sat Apr 29 17:41:45 2006 -0400
@@ -435,7 +435,7 @@ int _snd_rawmidi_virtual_open(snd_rawmid
 
 	seq_mode = 0;
 	if (mode & SND_RAWMIDI_NONBLOCK)
-		seq_mode |= O_NONBLOCK;
+		seq_mode |= SND_SEQ_NONBLOCK;
 
 	if (! slave_str)
 		slave_str = "default";
